src/test/prevector_tests.cpp

```diff
@@ -208,84 +208,85 @@ class prevector_tester {
         BOOST_CHECK_MESSAGE(passed, "insecure_rand: " + rand_seed.ToString());
     }
 
-    prevector_tester() {
-        SeedRandomForTest();
-        rand_seed = InsecureRand256();
+    prevector_tester(FastRandomContext& rng) {
+        SeedRandomStateForTest(SeedRand::SEED);
+        rng.Reseed(GetRandHash());
+        rand_seed = rng.rand256();
         rand_cache.Reseed(rand_seed);
```

Comment:
`rand_cache` is dead code since 16329224e70d0525208f6b0ba00c5e1531a4f5ea. Maybe remove it in an early clean-up commit?